osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/Device.java

```diff
@@ -403,12 +403,6 @@ public void setDeviceModel(final DeviceModel deviceModel) {
     this.deviceModel = deviceModel;
   }
 
-  public void addFirmwareFile(final FirmwareFile firmwareFile, final String installedBy) {
```

Comment:
If there is no `addFirmwareFile` anymore, is there any reason to keep `deviceFirmwareFiles` as a list? Where is this list used?

Reply:
The relation is there to keep track of the history of firmware files.... Bad naming, but we are not using it in SMHE, but other applications could use it!